### PR TITLE
Create a no data state component

### PR DESCRIPTION
**Pull request by caylahamann**
Screen Shot 2021-05-27 at 4 45 54 PM

### CODE REVIEW

File: src/no-data-state/styles.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  justify-items: center;
+  text-align: left;
+}
+
+.NoDataState-headingText {
+  color: #999;
```
Comment (jerelmiller) [inline src/no-data-state/styles.scss:17]: Is there an opportunity here to use one of our [color variables](https://github.com/newrelic/nr1-victory-visualizations/blob/main/src/styles.scss)?

File: src/no-data-state/styles.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+.NoDataState-bodyText {
+  color: #999;
```
Comment (jerelmiller) [inline src/no-data-state/styles.scss:21]: Same here. Could we use one of our color variables?

**Review by jerelmiller — COMMENTED** [review]
Added a very minor comment here. This is super helpful!

File: src/no-data-state/styles.scss (hunk 1 above)
Comment (caylahamann) [reply src/no-data-state/styles.scss:17]: we could! I just took the values from the in product visualizations, which uses different coloring than the nr1 sdk. We can just change it to a similar color

**Review by jerelmiller — APPROVED** [review]

**Pull request merged**